doc/whats_new.rst

```diff
@@ -45,6 +49,8 @@ API
 
 - New API for network creation. The default network is now created with ``net = default_network(params)``, by `Nick Tolley`_ in `#318 <https://github.com/jonescompneurolab/hnn-core/pull/318>`_
 
+- New API for calculating extracellular potentials using electrode arrays, created using :func:`~hnn_core.Network.add_electrode_array` and stored under ``net.rec_array`` as a dictionary of :class:`hnn_core.extracellular.ExtracellularArray` containers, by `Mainak Jas`_, `Nick Tolley`_ and `Christopher Bailey`_ in `#329 <https://github.com/jonescompneurolab/hnn-core/pull/329>`_
```

Comment:
API section is usually reserved for "changes" to the API. It's a bit of a misnomer. I would add this to the Changelog section as well.

Reply:
"As well" or "instead of"? Shall I move this and the previous 318 up to Changelog?

Reply:
I meant ... along with the entry for visualizing LFP, add this below :) So, yep, let's move this. 318 can be fixed later .... let's not mix other stuff in this PR